mupen64plus-core/upstream/src/plugin/plugin.c

```diff
@@ -69,6 +69,8 @@ static const gfx_plugin_functions dummy_gfx = {
     dummyvideo_ReadScreen2,
     dummyvideo_SetRenderingCallback,
     dummyvideo_ResizeVideoOutput,
+    dummyvideo_PluginResolutionReset,
```

Comment:
Hmm, could this be done with ResizeVideoOutput instead of expanding the API?

Reply:
I don't believe so, I tried using that function when I was first trying to get the resolution setting to work and I can't remember what the problem was; but the resolution reset function pauses the game after it gets a readable frame from the video plugin when it starts which I applied to settings I couldn't get to work without restarting the activity, I'll look into some other solutions that maybe don't involve the video plugins